tutorials/katacoda/thanos/2-lts/step3.md

```diff
@@ -0,0 +1,65 @@
+# Step 3 - Installing the Thanos Store
+
+In this step, we will learn about Thanos Store Gateway, how to start, and what problems are solved by it.
+
+## Thanos Components
+
+Thanos is a single Go binary capable to run in different modes. Each mode represents a different component and can be invoked in a single command.
+
+Let's take a look at all the Thanos commands:
+
+```docker run --rm quay.io/thanos/thanos:v0.13.0 --help```{{execute}}
+
+You should see multiple commands that solve different purposes, block storage based long-term storage for Prometheus.
+
+In this step we will focus on thanos `store gateway`:
+
+```
+  store [<flags>]
+    Store node giving access to blocks in a bucket provider
+```
+
+## Store Gateway/ Store :
+
+* This component implements the Store API on top of historical data in an object storage bucket. It acts primarily as an API gateway and therefore does not need significant amounts of local disk space.
+* It joins a Thanos cluster on startup and advertises the data it can access.
+* It keeps a small amount of information about all remote blocks on the local disk and keeps it in sync with the bucket.
+This data is generally safe to delete across restarts at the cost of increased startup times.
+
+
+You can read more about [Store](https://thanos.io/components/store.md/) here.
+
+## Deployment
+
+Before deploying, we need to stop the running thanos `Sidecar` here, we can do so by running:
+
+```
+docker stop sidecar
```

Comment:
why?

Reply:
IMO, it doesn't matter whether it's running or stopped, and the reason I stopped here, because it was giving errors related to the HTTP port already in use.

Reply:
or maybe I should use different ports for all containers, ugh ℺